assets/app/views/browse/pod.html

```diff
@@ -32,6 +32,7 @@
                   <dd>{{pod.status.podIP || 'unknown'}}</dd>
                   <dt>Restart policy:</dt>
                   <dd>{{pod.spec.restartPolicy || 'Always'}}</dd>
+                  <annotations ng-if="pod.metadata.annotations" annotations="pod.metadata.annotations"></annotations>                  
```

Comment:
Carryover from code changes, it can be removed, the annotations directive handles it.